### PR TITLE
User guide

### PR DESCRIPTION
**Pull request by javenseow**
Updated user guide. Please review to see if any changes are needed.

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -59,39 +59,49 @@ Format: `help`
...
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/John Doe p/98765432 e/johnd@example.com b/C r/420 s/School of Computing c/Basketball`
+* `add n/Betsy Crowe c/Soccer e/betsycrowe@example.com b/A p/1234567 r/213 s/Business`
 
 === Listing all persons : `list`
 
 Shows a list of all persons in the address book. +
 Format: `list`
 
+=== Searching all persons under specified tag : `search`
```
Comment (pixelky) [inline docs/UserGuide.adoc:77]: Should it be under specified CCA instead

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -242,19 +382,46 @@ _{explain how the user can enable/disable data encryption}_
...
 * *Delete* : `delete INDEX` +
 e.g. `delete 3`
-* *Edit* : `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...` +
+* *Erase* : `erase CCA` +
+e.g. `erase basketball`
+* *Edit* : `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [c/CCA]...` +
 e.g. `edit 2 n/James Lee e/jameslee@example.com`
 * *Find* : `find KEYWORD [MORE_KEYWORDS]` +
 e.g. `find James Jake`
+* *Search* : `search TAG [MORE_TAGS]` +
```
Comment (pixelky) [inline docs/UserGuide.adoc:407]: Should this be CCA too

**Comment by pixelky** [comment]
Probably need to change the function parameters once we finalise them

File: docs/UserGuide.adoc (hunk 1 above)
Comment (javenseow) [reply docs/UserGuide.adoc:77]: i was thinking that search can be for other tags also like block or school and not just cca uh

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -12,13 +12,13 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level4
+:repoURL: https://github.com/CS2103-AY1819S1-W16-1/main/tree/master
 
 By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
```
Comment (pixelky) [inline docs/UserGuide.adoc]: Do we need to change this

Comment (javenseow) [reply docs/UserGuide.adoc]: I think we should but what month do I put and I don't know what licence is

**Pull request merged**